build.gradle

```diff
@@ -19,7 +19,7 @@ def compatibilityVersion = 1.6
 sourceCompatibility = compatibilityVersion
 targetCompatibility = compatibilityVersion
 group = 'com.bmuschko'
-version = '2.2.2'
+version = '2.2.3'
```

Comment:
Could you please revert the version change? I am doing that as part of the release process.